src/components/ButtonsGroup.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { filterAndSortItems } from '../utils';
+
+import Link from './Link';
+
+const ButtonsGroup = ({
+  data: { filter, sort, items: _i2 } = {},
+  className,
+  classNames,
+  items: _i
+}) => {
+  const items = filterAndSortItems(_i || _i2 || [], { filter, sort });
```

Comment:
Button groups can be easily organised using `sort` and `filter` (see the mining pools example)

This will be helpful in i18n alphabetical sorting